rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -1014,23 +1037,53 @@ def run(self):
                 _logger.error(e)
                 return 1
 
-        if self.gen_job_name in ('METRICS_INGESTION', 'AW4_ARRAY_WORKFLOW', 'AW4_WGS_WORKFLOW'):
+        if self.gen_job_name in (
+            'METRICS_INGESTION',
+            'AW4_ARRAY_WORKFLOW',
+            'AW4_WGS_WORKFLOW',
+            'AW5_ARRAY_MANIFEST',
+            'AW5_WGS_MANIFEST'
+        ):
             try:
-                if self.args.manifest_file:
-                    _logger.info(f'File(s) Specified: {self.args.manifest_file}')
-                    return self.run_manifest_ingestion()
-
-                elif self.args.csv:
-                    _logger.info(f'File list Specified: {self.args.csv}')
-                    print(f'Multiple File List Specified: {self.args.csv}')
+                if self.args.manifest_file or self.args.csv:
+                    _logger.info(f'File(s) Specified: {self.args.manifest_file or self.args.csv}')
 
-                    # Validate file exists
-                    if not os.path.exists(self.args.csv):
-                        _logger.error(f'File {self.args.csv} was not found.')
+                    if self.args.csv and not os.path.exists(self.args.csv):
+                        _logger.error(f'CSV File {self.args.csv} was not found.')
                         return 1
 
-                    return self.process_multiple_from_file()
+                    file_paths = self.args.manifest_file if self.args.manifest_file else self.csv_to_list()
+                    bucket_name = file_paths.split('/')[0] if type(file_paths) is not list else file_paths[0].split(
+                        '/')[0]
+
+                    if self.args.cloud_task:
+                        # Get blob for file from gcs
+                        file_name = file_paths if type(file_paths) is not list else file_paths[0]
```

Comment:
I may be missing something, but if `file_paths` is a list, this looks like it only processes the first item in the list. So if the file referenced in `self.args.csv` has many lines, only the first manifest file is processed.